docs/decisions/index.md

```diff
@@ -11,6 +11,16 @@ architectural decision made along with its context and consequences. Further
 information can be found on Michael Nygard's
 [blog](https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions).
 
+The records in this site:
+
+- Should be read in conjunction, and linked, with the [central Hackney IT ADR repository](https://github.com/LBHackney-IT/lbh-adrs).
+- The records defined here should document Social Care Tools __specific variations__ from the [Hackney IT ADR decisions](https://github.com/LBHackney-IT/lbh-adrs).
+- Please ensure the Hackney Repository contains links to this repository.
+
+## Tools
+
+Architecture decision records [ADR] are generated and maintained using Nat Pryce's [adr-tools](https://github.com/npryce/adr-tools). As each ADR is plain text, they may be added without the tools - however the tools do help maintain consistency.
```

Comment:
Very minor: we've already noted that ADR stands for architectural decision record in the introduction of this page so I don't think it's needed here.

```suggestion
ADRs are generated and maintained using Nat Pryce's [adr-tools](https://github.com/npryce/adr-tools). As each ADR is plain text, they may be added without the tools - however the tools do help maintain consistency.
```

Reply:
Done - merci 👍🏻